addons/point_of_sale/models/pos_order.py

```diff
@@ -875,13 +875,13 @@ def get_sale_details(self, date_start=False, date_stop=False, config_ids=False,
                 # stop by default today 23:59:59
                 date_stop = date_start + timedelta(days=1, seconds=-1)
 
-            AND([domain,
+            domain = AND([domain,
                 [('date_order', '>=', fields.Datetime.to_string(date_start)),
                 ('date_order', '<=', fields.Datetime.to_string(date_stop))]
             ])
 
             if config_ids:
-                AND([domain, [('config_id', 'in', config_ids)]])
+                domain = AND([domain, [('config_id', 'in', config_ids.ids)]])
```

Comment:
`+=` ?

Reply:
Not need to, in the AND method domain is already there.

Reply:
indeed